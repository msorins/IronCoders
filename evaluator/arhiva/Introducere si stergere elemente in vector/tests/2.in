5
20 200 123456 14 991
3
0 4
0 2
1 1 20