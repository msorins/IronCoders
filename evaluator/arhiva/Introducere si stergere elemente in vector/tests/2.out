20 20 123456 991 